app/src/main/java/org/jellyfin/androidtv/util/InfoLayoutHelper.java

```diff
@@ -107,7 +107,7 @@ public static void addInfoRow(Context context, BaseItemDto item, int mediaSource
         if (includeRuntime) addRuntime(context, item, layout, includeEndTime);
         addSeriesStatus(context, item, layout);
         addRatingAndRes(context, item, mediaSourceIndex, layout);
-        addMediaDetails(context, audioStream, layout);
+        addMediaDetails(context, item, mediaSourceIndex, layout);
```

Comment:
Looks like the `audioStream` parameter is now unused so it can be removed (line 64)

Reply:
Removing it breaks line 54 and if removed it ends up with the same parameters as line 51.